### PR TITLE
Port eos tests to network_cli

### PR DESCRIPTION
**Pull request by Qalthos**
##### SUMMARY

See #32765

##### ISSUE TYPE

 - Tests Pull Request

##### ANSIBLE VERSION

```
2.5
```

### CODE REVIEW

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
OK, though technically the `provider:` lines should be removed

**Comment by Qalthos** [comment]
I left them in for running `connection=local` tests as well, even if not automated

File: test/runner/lib/manage_ci.py [hunk 1]
```diff
@@ -72,42 +72,50 @@ def __init__(self, core_ci):
...
-            try:
-                intercept_command(self.core_ci.args, cmd, 'ping', env=env)
-                return
-            except SubprocessError:
-                sleep(10)
-                continue
-
-        raise ApplicationError('Timeout waiting for %s/%s instance %s.' %
-                               (self.core_ci.platform, self.core_ci.version, self.core_ci.instance_id))
+        for tag in tags:
```
Comment (mattclay) [inline test/runner/lib/manage_ci.py]: Checking host connectivity with one connection type should be sufficient. This logic is only intended to verify the host is ready to accept tests, not to actually test any of the code.

Also, why does the connection type even need to be specified here? Is it now a required value?

**Review by mattclay — DISMISSED** [review]

**Review by trishnaguha — APPROVED** [review]

File: test/runner/lib/manage_ci.py (hunk 1 above)
Comment (Qalthos) [reply test/runner/lib/manage_ci.py]: Right, I understand what that does now... just swapping `local` for `network_cli` as appropriate now

**Review by gundalow — APPROVED** [review]

**Pull request merged**